### PR TITLE
Updated the template and readme

### PR DESCRIPTION
**Pull request by nik-redhat**
Updated the meeting minutes template and the readme so as to reflect the latest changes like the change of platform to google meet, removing NA meeting schedule, updated the calendar and also updated few meeting topics as they are no more useful or are generally empty.

Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
@Sheetalpamecha can you please review this.

File: README.md [hunk 1]
```diff
@@ -6,36 +6,28 @@ This repository is for management of all Gluster community initiatives. Come pla
 We host a community meeting every week, on Tuesdays. One week, it will be APAC
 friendly hours, and another it would be NA/EMEA region friendly hours.
 
-You can add [this calendar](https://calendar.google.com/calendar/b/1?cid=dmViajVibDBrbnNiOWQwY205ZWg5cGJsaTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) to get notification on Gluster Community.
+You can add [this calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDQ0YmRydTllMXYzdWFoMmpsbjdqNXJlYmNfMjAyMDEwMjdUMDkwMDAwWiBzYWptb2hhbUByZWRoYXQuY29t&tmsrc=sajmoham%40redhat.com&scp=ALL) to get notification on Gluster Community.
 
 APAC/EMEA Schedule -
```
Comment (Sheetalpamecha) [inline README.md:11]: Not required to mention specific regions. Schedule sounds good

File: template.md [hunk 2]
```diff
@@ -3,18 +3,14 @@
...
-- http://github.com/gluster/community
-- Recording of this meeting-
+- http://github.com/gluster/community/meetings
 
-### Date/Time: Check the [community calendar](https://calendar.google.com/calendar/b/1?cid=dmViajVibDBrbnNiOWQwY205ZWg5cGJsaTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+### Date/Time: Check the [community calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDQ0YmRydTllMXYzdWFoMmpsbjdqNXJlYmNfMjAyMDEwMjdUMDkwMDAwWiBzYWptb2hhbUByZWRoYXQuY29t&tmsrc=sajmoham%40redhat.com&scp=ALL)
 
 ### Bridge
 * APAC/EMEA friendly hours
   - Tuesday {Date}, 02:30PM IST
```
Comment (Sheetalpamecha) [inline template.md:12]: Tuesday/Thursday

File: template.md [hunk 3]
```diff
@@ -3,18 +3,14 @@
...
 
-- http://github.com/gluster/community
-- Recording of this meeting-
+- http://github.com/gluster/community/meetings
 
-### Date/Time: Check the [community calendar](https://calendar.google.com/calendar/b/1?cid=dmViajVibDBrbnNiOWQwY205ZWg5cGJsaTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+### Date/Time: Check the [community calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDQ0YmRydTllMXYzdWFoMmpsbjdqNXJlYmNfMjAyMDEwMjdUMDkwMDAwWiBzYWptb2hhbUByZWRoYXQuY29t&tmsrc=sajmoham%40redhat.com&scp=ALL)
 
 ### Bridge
 * APAC/EMEA friendly hours
```
Comment (Sheetalpamecha) [inline template.md:11]: Not required to mention specific regions.

**Review by Sheetalpamecha — CHANGES_REQUESTED** [review]

**Comment by Sheetalpamecha** [comment]
In template.md Under metrics section we can remove "New Bugs in last 14 days" and "Total Bugs" rows

File: template.md (hunk 2 above)
Comment (nik-redhat) [reply template.md:12]: Do we have meetings on Thursday's?

**Comment by nik-redhat** [comment]
Will make the changes.

File: template.md (hunk 2 above)
Comment (Sheetalpamecha) [reply template.md]: yes Tuesdays right. I got confused

**Review by Sheetalpamecha — APPROVED** [review]

**Pull request merged**